pkg/kademlia/kademlia_test.go

```diff
@@ -480,10 +480,6 @@ func TestClosestPeer(t *testing.T) {
 			chunkAddress: swarm.MustParseHexAddress("5000000000000000000000000000000000000000000000000000000000000000"), // 0101, want peer 1
 			expectedPeer: 1,
 		},
-		{
```

Comment:
should this be removed? it seems like this functionality is still there and should be tested no?

Reply:
Yes, of course, I missed this to put back. Thanks.